package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/UrpShaderImporter.cs

```diff
@@ -0,0 +1,178 @@
+
+
+using UnityEngine;
+
+namespace Unity.Formats.USD
+{
+    public class UrpShaderImporter : ShaderImporterBase
+    {
+                private static Material ChannelCombinerMat;
+
+        public UrpShaderImporter(Material material) : base(material)
+        {
+        }
+
+        public override void ImportFromUsd()
+        {
+            Material mat = Material;
+
+            if (DiffuseMap)
+            {
+                Debug.Log("here");
+                mat.SetTexture("_BaseMap", DiffuseMap);
+                mat.SetColor("_BaseColor", Color.white);
+            }
+            else
+            {
+                mat.SetColor("_BaseColor", Diffuse.GetValueOrDefault(mat.color));
+            }
+
+            // TODO: What about opacity map?
+
+            if (!IsSpecularWorkflow)
+            {
+                // Robustness: It would be ideal if this parameter were provided by HDRP, however that
+                // would require this asset package having a dependency on the HDRP package itself,
+                // which is (yet) not desirable.
+                mat.SetFloat("_WorkflowMode", /*Standard Metallic*/ 1);
+            }
+            else
+            {
+                mat.SetFloat("_WorkflowMode", /*Spec Color*/ 0);
+                // mat.EnableKeyword("_MATERIAL_FEATURE_SPECULAR_COLOR");
+                // mat.EnableKeyword("_SPECULARCOLORMAP");
```

Comment:
The other pipelines need special keywords to activate the workflow, is that not true for URP?

If not, maybe just delete these lines?